src/plugins/notebook/components/Notebook.vue

```diff
@@ -238,6 +239,34 @@ export default {
                 page
             };
         },
+        deleteEntry(entryId) {
+            const self = this;
+            const entryPos = getEntryPosById(entryId, this.internalDomainObject, this.selectedSection, this.selectedPage);
+            if (entryPos === -1) {
+                return;
+            }
```

Comment:
If this ever happens, we should provide a more descriptive error. Doing nothing will be a little confusing